viewing_party/party.py

```diff
@@ -0,0 +1,125 @@
+def create_movie(movie_title, genre, rating):
+    if movie_title is None:
+        return None
+    if genre is None:
+        return None
+    if rating is None:
+        return None
```

Comment:
we can put this all together into a compound conditional statement:

```suggestion
    if movie_title is None or genre is None or rating is None:
        return None
```